src/components/Home.js

```diff
@@ -13,8 +13,42 @@ import '../css/Home.css';
 import Button from 'react-bootstrap/Button';
 import Carousel from 'react-bootstrap/Carousel';
 import TextLoop from "react-text-loop";
+// import { useLocation } from "@reach/router";
```

Comment:
We should definitely remove this commented out code before committing it into our codebase!